openupgrade_scripts/scripts/base/16.0.1.3/pre-migration.py

```diff
@@ -12,6 +12,15 @@
 _logger = logging.getLogger(__name__)
 
 
+def enable_coupon_sharing_within_entity(cr):
+    """Check before merging `coupon_commercial_partner_applicability` into
+    `loyalty_partner_applicability` if it was installed in v15 to set the parameter
+    to True to keep the same functionality"""
+    if openupgrade.is_module_installed(cr, "coupon_commercial_partner_applicability"):
+        # The value of the configuration parameter is set to True.
+        cr["ir.config_parameter"].sudo().set_param("allow_coupon_sharing", "True")
```

Comment:
Please revert to the previous SQL, which was correct. It has been my fault.

Reply:
It was the silly test of the day and it slipped in, sorry.